swagger2.0/oic.r.cloudopenapi.swagger.json

```diff
@@ -2,10 +2,13 @@
   "swagger": "2.0",
   "info": {
     "title": "OCF Cloud Open API",
-    "version": "0.0.1"
+    "version": "0.4.0",
```

Comment:
What we've done for the version field is simply put in the date of the most recent change.  Pick your own date format.  Trying to do actual version style fields ends up with inconsistent content.